src/app/customer/Customer.tsx

```diff
@@ -82,7 +91,9 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, Cust
                 { isEmailLoginFormOpen && this.renderEmailLoginFormForm() }
                 { (viewType === CustomerViewType.Guest) ?
                     this.renderGuestForm() :
-                    this.renderLoginForm() }
+                    viewType === CustomerViewType.CreateAccount ?
```

Comment:
Can we pull this `viewType` check out into a private method or something? It feels odd to have two ternary conditions chained together.